### PR TITLE
TinyMCE lang

### PR DESCRIPTION
**Pull request by ebrehault**
- Use `i18n.currentLanguage` to initialise TinyMCE lang option,
- Fallback to closest lang if the required one is missing in TinyMCE (for instance, if `fr_be.js` is missing, we try `fr.js` and if `fr.js` is missing, we try `fr_Fr.js`)

### CODE REVIEW

**Comment by ebrehault** [comment]
@bloodbare, have a look when you have a moment please.

File: mockup/patterns/tinymce/pattern.js [hunk 1]
```diff
@@ -310,35 +350,34 @@ define([
...
+          self.options.scales = _.map(self.options.scales.split(','), function(scale){
+            var scale = scale.split(':');
+            return {
+              part: scale[1],
+              name: scale[1],
+              label: scale[0]
+            }
+          });
+        }
+        if(typeof(self.options.folderTypes) === 'string'){
```
Comment (thet) [inline mockup/patterns/tinymce/pattern.js]: shouldn't that and the following code be outside the current block and not within ``initLanguage``?

**Comment by vangheem** [comment]
Maybe bad merge? This might need to be updated with master.

**Comment by ebrehault** [comment]
@thet @vangheem we need to be done into a callback because `initLanguage` makes an ajax call to the lang file.

**Comment by thet** [comment]
@ebrehault ah, i see. it's a callback function. trying out locally....

**Pull request merged**